src/docs/components/docs-nav/docs-nav.css

```diff
@@ -68,24 +65,40 @@
     }
   }
   .docs-nav__summary--introduction,
+  .docs-nav__summary--introduction + ul .docs-nav__summary,
+  .docs-nav__summary--globals,
+  .docs-nav__summary--globals + ul .docs-nav__summary,
+  .docs-nav__summary--components,
+  .docs-nav__summary--components + ul .docs-nav__summary {
+    .docs-nav__summary-toggle {
+      display: block;
+    }
+  }
```

Comment:
```suggestion
  .docs-nav__summary {
    .docs-nav__summary-toggle {
      display: block;
    }
  }
```

Reply:
Alternatively, I would move the `display: block;` down and have it 3x the same - still less css written. ^^